docs/_themes/clusterhq/static/docs.css

```diff
@@ -0,0 +1,381 @@
+li.button.menu a {
+    font-family: lato;
+    border: 1px solid #ffffff;
+    background-color: transparent;
+    display: inline-block;
+    text-align: center;
+    line-height: 38px;
+    font-size: 1.125em;
+    margin-top: -6px;
+    border-radius: 8px;
+    float:right;
+    position:relative;
+    padding: 0 15px 0px 15px;
+}
+
+li.button.menu a:hover {
+    color: #a2dad7; 
+    border: 1px solid #a2dad7;
+}
+
+/* End of Nav styles */
+
+.menubar {
+    background: #1e2021; /* Old browsers */
+    /* IE9 SVG, needs conditional override of 'filter' to 'none' */
+    background: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/Pgo8c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjEwMCUiIGhlaWdodD0iMTAwJSIgdmlld0JveD0iMCAwIDEgMSIgcHJlc2VydmVBc3BlY3RSYXRpbz0ibm9uZSI+CiAgPGxpbmVhckdyYWRpZW50IGlkPSJncmFkLXVjZ2ctZ2VuZXJhdGVkIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgeDE9IjAlIiB5MT0iMCUiIHgyPSIxMDAlIiB5Mj0iMTAwJSI+CiAgICA8c3RvcCBvZmZzZXQ9IjAlIiBzdG9wLWNvbG9yPSIjMWUyMDIxIiBzdG9wLW9wYWNpdHk9IjEiLz4KICAgIDxzdG9wIG9mZnNldD0iMTAwJSIgc3RvcC1jb2xvcj0iIzM0M2EzZCIgc3RvcC1vcGFjaXR5PSIxIi8+CiAgPC9saW5lYXJHcmFkaWVudD4KICA8cmVjdCB4PSIwIiB5PSIwIiB3aWR0aD0iMSIgaGVpZ2h0PSIxIiBmaWxsPSJ1cmwoI2dyYWQtdWNnZy1nZW5lcmF0ZWQpIiAvPgo8L3N2Zz4=);
+    background: -moz-linear-gradient(-45deg, #1e2021 0%, #343a3d 100%); /* FF3.6+ */
+    background: -webkit-gradient(linear, left top, right bottom, color-stop(0%,#1e2021), color-stop(100%,#343a3d)); /* Chrome,Safari4+ */
+    background: -webkit-linear-gradient(-45deg, #1e2021 0%,#343a3d 100%); /* Chrome10+,Safari5.1+ */
+    background: -o-linear-gradient(-45deg, #1e2021 0%,#343a3d 100%); /* Opera 11.10+ */
+    background: -ms-linear-gradient(-45deg, #1e2021 0%,#343a3d 100%); /* IE10+ */
+    background: linear-gradient(135deg, #1e2021 0%,#343a3d 100%); /* W3C */
+    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#1e2021', endColorstr='#343a3d',GradientType=1 ); /* IE6-8 fallback on horizontal gradient */
+}
+
+.menubar .logo {
+    padding-top: 25px;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family:'Soho Gothic W01 Medium';
+    padding-bottom: 25px;
+}
+
+a.headerlink {
+    visibility: hidden;
+}
+
+h1:hover > a.headerlink, h2:hover > a.headerlink, h3:hover > a.headerlink, h4:hover > a.headerlink, h5:hover > a.headerlink, h6:hover > a.headerlink, dt:hover > a.headerlink {
+    visibility: visible;
+}
+
+.contents-menu {
+    background-color: #646570;
+}
+
+.contents-menu>ul {
+    list-style: none;
+    padding: 0;
+    font-size: 1.4em;
+    font-family: lato;
+    color: #FFFFFF;
+    width: 100%;
+    display: none;
+}
+
+.contents-menu>ul  a {
+    text-decoration: none;
+    color: #ffffff;
+}
+
+.contents-menu>ul  a:hover {
+    color: #9ac4c4;
+}
+
+.contents-menu>ul >li {
+    border-top: solid #a2dad7 1px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+}
+
+.contents-menu>ul >li:first-child {
+    border-top: none;
+}
+
+.contents-menu>ul ul>li {
+    font-size: 14px;
```

Comment:
We should at least be internally consistent with units used for the same things (e.g. all `font-size` declarations are `em`).

It would be even better to have a policy. My preference would be to copy Bootstrap's policy.